Client/src/Pages/Auth/hooks/useValidatePassword.jsx

```
@@ -0,0 +1,70 @@
+			path: error.path[0],
+			message: error.message,
+		}));
+		const errorsByPath =
+			errors &&
+			errors.reduce((acc, { path, message }) => {
+				if (!acc[path]) {
+					acc[path] = [];
+				}
+				acc[path].push(message);
+				return acc;
+			}, {});
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)